### PR TITLE
Cleanup & document a11y migration

### PR DESCRIPTION
**Pull request by shilman**
Issue: #5832 

## What I did
- simplify, document, test
- added a story to test story-level config
- renamed `withA11Y` to `withA11y` for consistency

## How to test

See new story in `official-storybook`

### CODE REVIEW

File: MIGRATION.md [hunk 1]
```diff
@@ -204,7 +205,31 @@ addParameters({ viewport: options });
...
+
+You should replace it with:
+
+```js
+import { addParameters } from '@storybook/react'; // or others
+
+addParameters({ a11y: options });
+```
+
+You can pass `a11y` parameters at the global level (via `addParameters` imported from `@storybook/react` et al.), and the story level (via the third argument to `.add()`).
```
Comment (tmeasday) [inline MIGRATION.md]: Should this line be "the component level"? (given the example above is global)

**Review by tmeasday — APPROVED** [review]
Thanks @shilman! Small suggestions

**Comment by Hypnosphi** [comment]
> renamed `withA11Y` to `withA11y`

I understand that `withA11Y` was only introduced in alpha and it's technicaly no breaking change, but It was really hard to debug the reason of "decorator is not a function" error =(

Can we keep the old spelling as a deprecated export?

**Comment by shilman** [comment]
Ugh, sorry about that @Hypnosphi. Will fix now!

**Pull request merged**